src/app/examples/form/form.model.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpErrorResponse } from '@angular/common/http';
+
+export interface Form {
+  firstname: string;
+  lastname: string;
+  email: string;
+  description: string;
+  wannagift: boolean;
+  age: number;
+  dob: Date;
+  address: {
```

Comment:
As mentioned previously, does our form use all of these fields?

Reply:
Yes it does. But the model is not really used as it should be to replace `any` in the effect.

Do you see a way to improve the loop getting data from the local storage using the `Form` type?
Then what is the cleanest way to map the `FormGroup` to a `Form` before dispatching it?